python/nammu/controller/ConsoleController.py

```diff
@@ -29,7 +29,20 @@ def __init__(self, mainControler):
         self.view = ConsoleView(self)
         # Will also need delegating to parent presenter
         self.controller = mainControler
+        # Need a record of previous messages so we can rebuild the html
+        self.console_record = []
 
     def addText(self, text):
-        self.view.edit_area.append(text)
-        self.view.scroll()
+        # Wrap the new console message in p tags and add it to the record
+        self.console_record.append('<p>{0}</p>'.format(text))
+
+        # Update the console with all of the messages, we cant just insert
+        # text as we have to insert within the <body> tags, so refreshing the
+        # console with a new html page is the best solution
+        self.view.edit_area.setText(''.join(self.console_record))
```

Comment:
I have tested the behaviour of this method, and when we have a long error message, the console scrolls to the last line without needing the call to `self.view.scroll()`.